### PR TITLE
Feature/protobuf

### PR DESCRIPTION
**Pull request by RohanDoshi21**
- Added ControlPlane and DataPlane,
- Tested it out with Malhar, working good with android

### CODE REVIEW

File: example/lib/src/views/vendor_models_ui/rgb_on_off/rgb_on_off_ui.dart [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        child: Column(
+          children: [
+            const Padding(
+              padding: EdgeInsets.all(15.0),
+              child: Text('ON-OFF'),
+            ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                // ElevatedButton(
```
Comment (IshanDaga) [inline example/lib/src/views/vendor_models_ui/rgb_on_off/rgb_on_off_ui.dart]: Don't leave commented out functionality / code in a PR
Remove dead code from here

File: example/lib/src/views/vendor_models_ui/rgb_on_off/rgb_on_off_ui.dart [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                //   } catch (e) {
+                //     scaffoldMessenger.showSnackBar(SnackBar(content: Text(e.toString())));
+                //   }
+                //   },
+                //   child: const Text('ON'),
+                // ),
+                const SizedBox(
+                  width: 20,
+                ),
+                // ElevatedButton(
```
Comment (IshanDaga) [inline example/lib/src/views/vendor_models_ui/rgb_on_off/rgb_on_off_ui.dart]: Remove dead code

File: lib/src/events/data/vendor_model_message_status/vendor_model_message_status.dart [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/foundation.dart';
```
Comment (IshanDaga) [inline lib/src/events/data/vendor_model_message_status/vendor_model_message_status.dart:1]: Comments missing in this file

**Review by IshanDaga — CHANGES_REQUESTED** [review]
### Tasks pending after review:
- [x] Remove dead code
- [x] In code comments and docs in marked files

File: example/lib/src/views/vendor_models_ui/rgb_on_off/rgb_on_off_ui.dart (hunk 1 above)
Comment (RohanDoshi21) [reply example/lib/src/views/vendor_models_ui/rgb_on_off/rgb_on_off_ui.dart]: This was used previously when our protobuf consisted of the enum `status`, to only transmit `on` or `off` 
Will remove it though in the new commit

**Review by IshanDaga — APPROVED** [review]

**Pull request merged**